Solutions/rahulkumaran/Task4/sin.py

```diff
@@ -0,0 +1,13 @@
+import matplotlib.pyplot as plt
+import numpy as np
+
+xRange = np.arange(0.0,2.0,0.01)
+sinGraph = np.sin(2*np.pi*xRange)
```

Comment:
It was `y = 2sin(x*pi/4)` Not `y = sin(2*pi*x)`. Edit it, rest looks good.

Reply:
Done :smile: